app/delivery/send_to_providers.py

```diff
@@ -284,13 +293,12 @@ def get_html_email_options(notification: Notification) -> Dict[str, Union[str, b
     return options_dict
 
 
-def technical_failure(notification):
-    notification.status = NOTIFICATION_TECHNICAL_FAILURE
-    dao_update_notification(notification)
-    raise NotificationTechnicalFailureException(
-        'Send {} for notification id {} to provider is not allowed: service {} is inactive'.format(
-            notification.notification_type, notification.id, notification.service_id
-        )
+def inactive_service_failure(notification: Notification):
+    # notification.status = NOTIFICATION_PERMANENT_FAILURE
+    # dao_update_notification(notification)
```

Comment:
let's not leave comments in running code without an explanation; please remove or explain why these are here